client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main-deprecated.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button } from '@automattic/components';
+import { __experimentalUseNavigator as useNavigator } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import NavigatorHeader from './navigator-header';
+import PatternLayout from './pattern-layout';
+import type { Pattern } from './types';
+
+interface Props {
+	sections: Pattern[];
+	header: Pattern | null;
+	footer: Pattern | null;
+	onAddSection: () => void;
+	onReplaceSection: ( position: number ) => void;
+	onDeleteSection: ( position: number ) => void;
+	onMoveUpSection: ( position: number ) => void;
+	onMoveDownSection: ( position: number ) => void;
+	onAddHeader: () => void;
+	onDeleteHeader: () => void;
+	onAddFooter: () => void;
+	onDeleteFooter: () => void;
+	onContinueClick: () => void;
+}
+
+const ScreenMainDeprecated = ( {
```

Comment:
The current layout of the sidebar

![image](https://user-images.githubusercontent.com/13596067/220594565-cad8ee45-188a-4f3e-8e9e-a4c09bd56024.png)